examples/data-solutions/dp-foundation/01-lnd-datastorage.tf

```diff
@@ -0,0 +1,93 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+locals {
+  lnd_bucket_retention_policy = {
+    retention_period = 7776000 # 90 * 24 * 60 * 60
+    is_locked        = false
+  }
+}
+###############################################################################
```

Comment:
Lore, can you switch to single line comments? We don't use big blocks anywhere in fast